cp-overrides.yaml

```diff
@@ -5,15 +5,15 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-27
+                  ref: sprint-28
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_720
+                  ref: EAP_721
```

Comment:
@rgdoliveira @apintea @luck3y Hey guys, where the ```EAP_721``` [jboss-eap-modules](https://github.com/jboss-container-images/jboss-eap-modules.git) tag, release, or branch, referenced above can be found?

I can see / find the ```EAP_721``` tag for [jboss-eap-7-image](https://github.com/jboss-container-images/jboss-eap-7-image/releases) repository, but can't find neither ```EAP_721``` tag, nor release, nor branch for the [jboss-eap-modules](https://github.com/jboss-container-images/jboss-eap-modules/tags) repo. Should this be created yet?

Thank you! Jan

Reply:
@iankko Since this is a CP, we wouldn't normally be making new tags. In this case the EAP_721 tag would be an exact copy of the sprint-28 tag.

Reply:
@luck3y Thanks, Ken! So I am going to use the ```sprint-31``` tag in RH-SSO OpenShift image then.